dasharo-compatibility/dasharo-tools-suite.robot

```diff
@@ -0,0 +1,150 @@
+*** Settings ***
+Library             Collections
+Library             OperatingSystem
+Library             Process
+Library             String
+Library             Telnet    timeout=20 seconds    connection_timeout=120 seconds
+Library             SSHLibrary    timeout=90 seconds
+Library             RequestsLibrary
+Resource            ../keywords.robot
+Resource            ../keys.robot
+Resource            ../rtectrl-rest-api/rtectrl.robot
+Resource            ../sonoff-rest-api/sonoff-api.robot
+Resource            ../variables.robot
+
+Suite Setup         Run Keywords
+...                     Prepare Test Suite
+...                     AND
+...                     Enable Network Boot
+...                     AND
+...                     Disable Secure Boot
+...                     AND
+...                     Upload And Mount DTS Flash ISO
+Suite Teardown      Run Keyword
+...                     Log Out And Close Connection
+
+
+*** Test Cases ***
+DTS001.001 Booting DTS from USB works correctly
+    [Documentation]    This test aims to verify that DTS is properly booting
+    ...    from USB.
+    Skip If    not ${TESTS_IN_FIRMWARE_SUPPORT}    DTS001.001 not supported
+    Skip If    not ${DTS_SUPPORT}    DTS001.001 not supported
+
+    Power On
+    Boot Dasharo Tools Suite    USB_emulated
+
+DTS002.001 DTS option Creating Dasharo HCL report works correctly
+    [Documentation]    This test aims to verify that the option Dasharo HCL
+    ...    report in the DTS menu properly creates the report.
+    Skip If    not ${TESTS_IN_FIRMWARE_SUPPORT}    DTS002.001 not supported
+    Skip If    not ${DTS_SUPPORT}    DTS002.001 not supported
+    # Not supported due to SSH authorization error while sending the report
+    Skip
+    Power On
+    Boot Dasharo Tools Suite    iPXE
+    Check HCL Report Creation
+
+DTS003.001 DTS option power-off DUT works correctly
+    [Documentation]    This test aims to verify that the option Power off
+    ...    system in the DTS menu turns off the DUT.
+    Skip If    not ${TESTS_IN_FIRMWARE_SUPPORT}    DTS003.001 not supported
+    Skip If    not ${DTS_SUPPORT}    DTS003.001 not supported
+
+    Power On
+    Boot Dasharo Tools Suite    iPXE
+    Sleep    3s
+    Write Into Terminal    10
+    Check Power Off In DTS
```

Comment:
Does it make sense to have kwd that is used only once?
When we generate test case documentation from this, it would be better if the content of the `Check Power Off In DTS` would be directly here

Reply:
[Keyword removed](https://github.com/Dasharo/open-source-firmware-validation/pull/93/commits/c25cb497706d8a74c04839a38cf75f2ad65e1a61#diff-4d0e5cbb7f43565e1c6904f6f0c660951a6f60f7d187a787aa23d8fcab380777R58)